kinder/doc/kind-kinder.md

```diff
@@ -2,7 +2,9 @@
 
 [kind](https://github.com/kubernetes-sigs/kind) is awesome!
 
-kinder is an example of [kind](https://github.com/kubernetes-sigs/kind) used as a library.
+kinder shares the same core ideas o kind, but, in agreement with the kind team, was developed
```

Comment:
kinder shares the same core ideas o kind, but, in agreement with the kind team, was developed
->
kinder shares the same core ideas of kind, but in agreement with the kind team, was developed